packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/add-notebook.html

```diff
@@ -14,14 +14,13 @@
                     data-translate="data_processing_add_notebook_description_warning_ai"
                 >
                 </span>
-                <a
-                    class="oui-link oui-link_icon"
-                    data-ng-href="{{:: $ctrl.aiNotebookLink()}}"
-                >
-                    <span
-                        data-translate="data_processing_add_notebook_description_warning_ai_link"
-                    ></span>
-                </a>
+                <span>
+                    <a href="" data-ng-click="$ctrl.trackAndGoToAiNotebook()">
+                        <span
+                            data-translate="data_processing_add_notebook_description_warning_ai_link"
+                        ></span>
+                    </a>
```

Comment:
You have to use `oui-button` instead of having anchor element with an empty `href`.

Reply:
Unfortunately, I already tried to use a oui-button (with data-variant="link") here but it ends up not in line with the rest of the text, since it's a link at the end of a sentence.

Reply:
for me the only situation where the `href` is empty is in case where it update dynamically within the JavaScript.
having an empty anchor produce accessibility issue.

if your issue is to inline horizontally the element you can use the bootstrap CSS helpers classes.